cmake/ccf_app.cmake

```diff
@@ -186,7 +186,11 @@ function(add_ccf_app name)
     )
 
     if(NOT SAN)
-      target_link_options(${virt_name} PRIVATE LINKER:--no-undefined)
+      target_link_options(
+        ${virt_name}
+        PRIVATE
+        LINKER:--no-undefined,--undefined=enclave_create_node,--undefined=enclave_run
```

Comment:
These symbols are defined in `enclave/main.cpp`, which gets built into `libccf.virtual.a` and `libccf.enclave.a`, which then get linked to by CCF apps into libraries like `liblogging.virtual.so`. Something in this branch, though I don't know what, changed the behaviour of these symbols, and they were no longer exported from the combined `.so`. In classic fashion, I'm now not sure why this _ever_ worked without this line, which tells the linker to explicitly re-export those symbols from the shared object, even though they're not used _by_ the shared object. It works now, afaict.